src/Symfony/Component/Notifier/Notification/Notification.php

```diff
@@ -31,18 +31,13 @@ class Notification
         LogLevel::EMERGENCY => 600,
     ];
 
-    public const IMPORTANCE_URGENT = 'urgent';
-    public const IMPORTANCE_HIGH = 'high';
-    public const IMPORTANCE_MEDIUM = 'medium';
-    public const IMPORTANCE_LOW = 'low';
-
-    private $channels = [];
-    private $subject = '';
-    private $content = '';
-    private $emoji = '';
-    private $exception;
-    private $exceptionAsString = '';
-    private $importance = self::IMPORTANCE_HIGH;
+    protected $channels = [];
+    protected $subject = '';
```

Comment:
why making them protected ? this increases the maintenance burden, and it does not unlock any new possibilities. child classes can already use the public getter and setter methods